Example/src/screens/StatusBar.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {useLayoutEffect, useState} from 'react';
+import {View, StyleSheet, Platform, Text} from 'react-native';
+import {enableScreens} from 'react-native-screens';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationProp,
+} from 'react-native-screens/native-stack';
+import {SettingsPicker, SettingsSwitch, Button, Spacer} from '../shared';
+
+enableScreens();
+
+type StackParamList = {
+  Main: undefined;
+};
+
+interface MainScreenProps {
```

Comment:
I think it would be nice to have 2 same screens here, to show the animation of status bar when moving between screens.